client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -1,39 +1,129 @@
 @import '@wordpress/base-styles/mixins';
 @import '@wordpress/base-styles/breakpoints';
+@import '@automattic/typography/styles/variables';
 
-.step-container.launchpad {
-	max-width: 1440px;
-    padding-inline-start: 24px;
-    padding-inline-end: 24px;
-
-	@include break-small {
-		padding-inline-start: 48px;
-    	padding-inline-end: 48px;
-	}
-	
-	.spinner-line {
+// Launchpad - Layout
+// This is styling common parts of stepper page
+.launchpad {
+	.progress-bar,
+	.step-container__header {
 		display: none;
 	}
+	.step-container {
+		background: var( --studio-gray-0 );
+		padding: 0;
+		margin: 0;
+		max-width: none;
+		min-height: 100vh;
+		.step-container__navigation.action-buttons {
+			top: auto;
+			inset-inline-start: 40px;
+			.step-container__skip-wrapper {
+				margin-inline-start: 0;
+			}
+		}
+	}
 }
 
 .launchpad__content {
 	display: flex;
 	flex-direction: column;
-	justify-content: center;
-	transform: translateY( -48px );
+	@include break-large {
+		flex-direction: row;
+	}
+}
+
+// Launchpad - Sidebar
+
+.launchpad__sidebar {
+	width: 440px;
+	padding: 20px 40px;
+	margin: 0 auto;
+	@include break-large {
+		margin: 0;
+	}
+}
+
+.launchpad__sidebar-h1 {
```

Comment:
Maybe we can use a more descriptive class name like 'launchpad__sidebar-header' ? No big deal for this PR though.